### PR TITLE
Food Recall Button w/ Web View

### PR DESCRIPTION
**Pull request by jl-codes**
Current Home Screen:
Screen Shot 2019-08-08 at 4 09 25 PM

Current Webview (post-segue):
Screen Shot 2019-08-08 at 4 09 41 PM

### CODE REVIEW

**Comment by WoppleSupreme** [comment]
This looks good so far. Is there anything in the code that would be problematic with merging with other people's code?

**Comment by WoppleSupreme** [comment]
There is core data functionality in this version which will not work on my machine when I try to run it. I didn't change anything, though. Probably just need to remove that functionality.

File: Recipick/ViewController.swift [hunk 1]
```diff
@@ -7,8 +7,19 @@
 //
 
 import UIKit
+import SafariServices
 
 class ViewController: UIViewController {
+  
+  @IBAction func recentRecalls(_ sender: UIButton) {
+    showSafariVC(for: "https://www.fda.gov/safety/recalls-market-withdrawals-safety-alerts")
```
Comment (prrenfrew) [inline Recipick/ViewController.swift]: I would suggest putting the URL into a constants namespace. You can use an enum or a struct for the namespace.

**Comment by jl-codes** [comment]
> This looks good so far. Is there anything in the code that would be problematic with merging with other people's code?

There shouldn't be, the only modifications explicitly done were in the ViewController and Storyboard

**Pull request closed without merging**